### ISSUE TITLE
Ensure that alerts that appear at the top of the page are announced to the screen reader user.

### ISSUE THREAD

**Issue by lhenze** [nyudlts/ultraviolet#146]
When creating an account or verifying an email address, alerts that appear at the top of the page are not announced to the screen reader user. They will need to navigate through the page until they come upon the information.
